packages/messages/src/browser/notification-preferences.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { interfaces } from "inversify";
+import {
+    createPreferenceProxy,
+    PreferenceProxy,
+    PreferenceService,
+    PreferenceContribution,
+    PreferenceSchema
+} from '@theia/core/lib/browser/preferences';
+
+export const NotificationConfigSchema: PreferenceSchema = {
+    "type": "object",
+    "properties": {
+        "notification.autoDismiss": {
```

Comment:
please rename it to something like `notification.timeout`. I'm planing to revise more parts of user notifications in #1360, and I think we should distinct between dismissing and hiding a message. a timeout is  more generics, which leaves more room for interpretation.

Reply:
just to mention it, hiding message and keeping them in kind of an inbox are ideas from #1360. I'm aware of this PR to target the closing of messages after the timeout period elapsed.

Reply:
auto-dismiss rename to timeout